GameServerLib/Logic/Content/ItemCollection.cs

```diff
@@ -6,57 +6,12 @@
 
 namespace LeagueSandbox.GameServer.Logic.Content
 {
-    public class ContentCollectionEntry
+    public class ContentCollectionEntry : ContentFile
```

Comment:
Not really relevant to this commit, but `ContentFile` isn't really the clearest of names. Something like `MetadataContainer`, `MetadataFile`, `ConfigFile` or similar would be better.